modules/terminate-agent-hook/main.tf

```diff
@@ -18,17 +18,17 @@ data "archive_file" "terminate_runner_instances_lambda" {
 # tfsec:ignore:aws-lambda-enable-tracing
 resource "aws_lambda_function" "terminate_runner_instances" {
   architectures    = ["x86_64"]
```

Comment:
Is see you fixing options, guess since the lambda is provided by us. Maybe good also to move to arm. Faster and cheaper. I moved all my Node lambda's last year to arm